new/LIT6166MiraclesJohnBaptist.xml

```diff
@@ -53,10 +55,170 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     </teiHeader>
     <text>
         <body>
+          <div type="edition">
+            <note>The text follows <ref type="mss" corresp="ESmy023"/>.</note>
+             <div type="textpart" subtype="chapter" xml:id="Introduction">
+                 <label>Introduction</label>
+                 <ab>
+                   <pb n="78ra"></pb>
```

Comment:
I wonder about the folio references in the work record - I think it might be better to limit them to the manuscript records. What do you think?

Reply:
For me it does not make any difference, it is more logical to record only the pure text. Editions do not consider the rubrication. Should I remove it? Perhaps a line on that in the guidelines would be helpful.